src/entrypoints/dead-clicks-autocapture.ts

```diff
@@ -157,6 +157,10 @@ class LazyLoadedDeadClicksAutocapture implements LazyLoadedDeadClicksAutocapture
             return true
         }
 
+        if (isDisabledElement(click.node)) {
```

Comment:
we captured a dead click after someone clicked on a disabled button, that feels wrong to me

Reply:
I can live with leaving it as is for now 👍